test/src/loaders/workspace_uploader/workspace_uploader_test.py

```diff
@@ -368,14 +423,159 @@ def test_generator(setup_and_teardown):
     assert assemblyTuple_list == expected_assemblyTuple_list
 
 
-def test_upload_assembly_files_in_parallel(setup_and_teardown):
+def test_upload_genome_files_in_parallel(setup_and_teardown):
     params = setup_and_teardown
+    collection_source_dir = params.collection_source_dir
+    sourcedata_dir = params.sourcedata_dir
     src_files = params.target_files
+    genbank_files = params.genbank_files
```

Comment:
```suggestion
    sourcedata_dir = params.sourcedata_dir
    # Source files here are the assembly files, which are normally produced by GFU.
    # This test fakes that process by placing them in the container job directory in the setup below
    assembly_files = params.target_files
    genbank_files = params.genbank_files
```

Reply:
👍

Reply:
Can you change the comment so it doesn't refer to source files any more? I should've fixed that when I changed src_files to assembly_files but I forgot

Reply:
👍

Reply:
:+1: